docs/community/SIGs/Runtime/SIG-Runtime-CHARTER.md

```diff
@@ -0,0 +1,144 @@
+# SIG Runtime — Charter
+
+**Status:** Proposed  
+**Last updated:** 2025-10-01
+
+## Mission & Scope
+
+### Mission
+
+The SIG Runtime maintains and shapes
+
+- **OCM Language Bindings** ([bindings](https://github.com/open-component-model/open-component-model/tree/main/bindings))
+- **OCM CLI** ([cli](https://github.com/open-component-model/open-component-model/tree/main/cli))
+- **OCM Kubernetes Controller** ([kubernetes/controller](https://github.com/open-component-model/open-component-model/tree/main/kubernetes/controller))
+
+The mission of SIG Runtime is to allow OCM Components to be
+
+- creatable
+- signable
+- transportable
+- verifiable
+- deployable
+
+### Scope
+
+- **OCM Language Bindings**
+  - Modular, extensible runtime library for components and versions
+  - Support for dynamic creation of component versions
+  - Standards-compliant signing and verification per the [OCM Specification](https://github.com/open-component-model/ocm-spec/tree/main)
+  - Introspection, transport, and transformation of components and versions
+
+- **OCM CLI**
+  - Unified toolchain to create, inspect, sign, verify, and transport components and versions
+  - Direct alignment with language bindings to ensure consistent semantics
+
+- **OCM Kubernetes Controller**
+  - Native Kubernetes API for OCM repositories, components, and artifacts
+  - Seamless integration with mainstream deployment frameworks (e.g., Flux, Kro)
+  - Controllers to synchronize, verify, and deploy OCM component versions within clusters
+
+- **Patterns and Best Practices**
+  - Authoring, curation, and dissemination of runtime deployment patterns
+  - Reference implementations, blueprints, and operational guidance for the community
+
+## Deliverables
```

Comment:
I feel that this section could use more juice - or we remove the bullet points markdown completely